x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_bulk_add_to_case_actions.tsx

```diff
@@ -32,7 +31,7 @@ export const useBulkAddToCaseActions = ({ onClose, onSuccess }: UseAddToCaseActi
   });
 
   return useMemo(() => {
-    return hasWritePermissions
+    return casePermissions.create && casePermissions.read
```

Comment:
For Stack cases, for a role with `Read` on Cases plus `Delete` checked, the bulk action for delete is not available:

![image](https://user-images.githubusercontent.com/7405507/178280417-3ad78660-f3c5-4633-ae90-1ba91c534b7f.png)

Only the delete action in the header of the case view page is available.